packages/flutter_test/lib/src/matchers.dart

```diff
@@ -224,6 +224,14 @@ Matcher isMethodCall(String name, {@required dynamic arguments}) {
   return new _IsMethodCall(name, arguments);
 }
 
+/// Asserts that 2 paths cover the same area by sampling multiple points.
+///
+/// Samples [numSamplesSqrt]^2 points inside [areaToCompare], and asserts that
+/// the [Path.contains] method returns the same value for each of the points
+/// for both paths.
+Matcher coversSameAreaAs(Path expectedPath, {@required Rect areaToCompare, int numSamplesSqrt = 20})
+  => new _CoversSameAreaAs(expectedPath, areaToCompare: areaToCompare, numSamplesSqrt: numSamplesSqrt); 
```

Comment:
we should avoid abbreviations like "num" and "sqrt" if possible. maybe just "sampleSize"?

Reply:
done